src/Microsoft.DotNet.ProjectModel/FileSystemGlobbing/Abstractions/DirectoryInfoBase.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
The copyright header needs to match the CLI's copyright header.  (i.e. MIT license, not Apache license)

I got permission from @Eilon to change the copyright for the DNX Command Line parser sources.  I'm assuming this code falls under the same umbrella, since it is the exact same scenario.